gymnasium_robotics/envs/adroit_hand/adroit_pen.py

```diff
@@ -277,31 +274,40 @@ def step(self, a):
             - self.data.site_xpos[self.tar_b_site_id]
         ) / self.tar_length
 
-        # pos cost
-        dist = np.linalg.norm(obj_pos - desired_loc)
-        reward = -dist * (not self.sparse_reward)
-        # orien cost
+        # compute the sparse reward variant first
+        goal_distance = np.linalg.norm(obj_pos - desired_loc)
         orien_similarity = np.dot(obj_orien, desired_orien)
-        reward += orien_similarity * (not self.sparse_reward)
+        goal_achieved = (
+            True if (goal_distance < 0.075 and orien_similarity > 0.95) else False
+        )
+        reward = 1.0 if goal_achieved else -0.1
+        goal_failed = obj_pos[2] < 0.075
```

Comment:
Remove this line, pre-commit fails because the variable is not used

Reply:
Kk